docusaurus/docs/getting-started/what-s-included/browser-support.md

```diff
@@ -0,0 +1,11 @@
+---
+id: browser-support
+title: Browser Support
+sidebar_label: Browser Support
+---
+
+By default, this boilerplate is configured to support the latest two versions of the most popular desktop and mobile browsers.
+
+You can find a list of what browsers this boilerplate supports [here](https://browserl.ist/?q=last%202%20Chrome%20versions%2C%20last%202%20Firefox%20versions%2C%20last%202%20Safari%20versions%2C%20last%202%20Edge%20versions%2C%20last%202%20ChromeAndroid%20versions%2C%20last%202%20iOS%20versions).
+
+This means that all the JS and CSS are transpiled to support the property `browserslist` of `package.json`.
```

Comment:
I think it would be better as:

"We're configuring our list of supported browsers in the `browserlist` propertry of our `package.json` file, which all JS and CSS files will be transpiled to support."

Also, transpiled vs compiled? Not sure, but i think compile is preferred elsewhere in the document.

Reply:
Changed to:

> We're configuring our list of supported browsers in the `browserlist` propertry of our `package.json` file. Feel free to change it in case your default browser support doesn't suit your project.